t/test.py

```diff
@@ -89,6 +89,14 @@ def test_init_accepts_instance_locale(self):
         date = jdatetime.date(1397, 4, 23, locale=jdatetime.FA_LOCALE)
         self.assertEqual(date.strftime('%A'), u'شنبه')
 
+    def test_strftime_fa_month(self):
+        j_months_fa = jdatetime.date.j_months_fa
+        month = 0
+        for date in range(12):
+            month += 1
+            j_month_fa = jdatetime.date(year=1400, month=month, day=10).strftime("%E")
+            self.assertEqual(j_month_fa, j_months_fa[month - 1])
```

Comment:
I would make it simpler:
```suggestion
        for month in range(1, 13):
            self.assertEqual(
                jdatetime.date(year=1400, month=month, day=10).strftime("%E"),
                jdatetime.date.j_months_fa[month - 1],
            )
```

Reply:
Thanks for the review, changes have been applied.